doc/verification.rst

```diff
@@ -0,0 +1,68 @@
+.. _Verification:
+
+Verification and Validation
+===========================
+This guide is inteded to be used to document major ``ADflow`` cases for validation and verification purposes.
+
+
+Overset and Multiblock Meshes
+-----------------------------
+The `following paper <https://arc.aiaa.org/doi/pdf/10.2514/1.C034486>`_ compares ``ADflow`` with OVERFLOW and elsA for the CRM and a Wing-Body-Nacelle-Pylon configuration. 
```

Comment:
I am guessing the bibtex entry below is for this paper? If so, can you make sure to put an identifier here ([1], or coder et al, or Coder2017a or whatever) so that people know this "paper" is the listing below. Also please change the link to the proper doi version, so instead of https://arc.aiaa.org/doi/pdf/10.2514/1.C034486 just do https://doi.org/10.2514/1.C034486 . The previous version should also be fine but I just like to keep the links consistent so that everything is rerouted through doi.org.

Reply:
Should we not use `sphinxcontrib-bibtex` here for citations?

Reply:
yeah if that is possible, then sure

Reply:
OK I will update the docs and set up the citations.

Reply:
This has been done